src/app/favourite/favourite.component.html

```diff
@@ -0,0 +1,3 @@
+<button (click)="addToFavorite()" [ngClass]="{'filled-btn':isLiked}" #favBtn class="favourite-btn">
```

Comment:
why don't you use recipe.saved instead of creating an unnecessary new variable since you already have it declared?

Reply:
and it's name isn't supposed to be isFavourite instead of isLiked if you want to use it?